examples/plot_knockoff_aggregation.py

```diff
@@ -54,7 +47,7 @@
 # number of repetitions for the bootstraps
 n_bootstraps = 25
 # number of jobs for repetition of the method
-n_jobs = 2
+n_jobs = 1
```

Comment:
- I think it makes sense to illustrate in the example where users should use parallelization and to do so using n_jobs>1 
 - If we decide not to show it I would remove n_jobs and rely on the default

Reply:
`n_jobs=2`  (or even 4) is a better option IMHO.

Reply:
What kind of issues ?

Reply:
May be related to #356. 
We still have the problem of nested parallel loops in the example generation. I would still argue that showcasing parallelization in the example is valuable.